pennylane/ops/qubit.py

```diff
@@ -2505,6 +2505,92 @@ def diagonalizing_gates(self):
         """
         return [QubitUnitary(self.eigendecomposition["eigvec"].conj().T, wires=list(self.wires))]
 
+# =============================================================================
+# Arithmetic
+# =============================================================================
+
+class Carry(Operation):
+    r"""Carry()
+    Apply a ``Carry`` operation on input wires
+    TODO add more information
+
+    **Details:**
+
+    * Number of wires: 4
+    * Number of parameters: 0
+    TODO consider gradient
+    * Gradient recipe: TODO
+
+    Args:
+        wires (Union[Wires or int]): the wires the operation acts on
```

Comment:
```suggestion
        wires (Sequence[int]): the wires the operation acts on
```
See https://github.com/PennyLaneAI/pennylane/pull/1127

Reply:
Thanks, useful to know!